lib/api/src/modules/stories.ts

```diff
@@ -144,12 +144,7 @@ export const init: ModuleFn = ({
     },
     getCurrentParameter: (parameterName) => {
       const { storyId, refId } = store.getState();
-      const parameters = api.getParameters({ storyId, refId }, parameterName);
-
-      if (parameters) {
-        return parameters;
-      }
-      return undefined;
+      return api.getParameters({ storyId, refId }, parameterName);
```

Comment:
This returns `undefined` when the parameter is `false`

Reply:
Unfortunately removing this caused various things to break, so I reverted the change. 😭